maintainers/maintainer-list.nix

```diff
@@ -6348,13 +6348,15 @@
     github = "hiljusti";
     githubId = 17605298;
   };
+
   hirenashah = {
     email = "hiren@hiren.io";
     github = "hirenashah";
     githubId = 19825977;
     name = "Hiren Shah";
   };
 
+
```

Comment:
This is unrelated changes and needs to be removed, or at least make it consistent with others.